jdbc/src/test/java/org/apache/iotdb/jdbc/IoTDBJDBCResultSetTest.java

```diff
@@ -233,6 +236,12 @@ public void testQuery() throws Exception {
               + "105,11.11,199,33333,11.11,\n"
               + "1000,1000.11,55555,22222,1000.11,\n"; // Note the LIMIT&OFFSET clause takes effect
       Assert.assertEquals(standard, resultStr.toString());
+      List<Object> standardObject = new ArrayList<>();
+      constructObjectList(standardObject);
+      Assert.assertEquals(standardObject.size(), resultObjectList.size());
+      for (int i = 0; i < standardObject.size(); i++) {
+        Assert.assertEquals(standardObject.get(i), resultObjectList.get(i));
+      }
```

Comment:
Hi, I don't think the testing way is ok because the assertEquals() method just compares their value. If `result.getObject(i)` returns a String, the test result is still right.

Reply:
<img width="1113" alt="Screen Shot 2021-03-02 at 2 42 29 PM" src="https://user-images.githubusercontent.com/25913899/109608907-92382b00-7b65-11eb-9934-0333764cc1b2.png">

If I change `result.getObject(i)` to `result.getString(i)`, the test failed....

Reply:
My fault...Sorry about that